neptune/new/attributes/atoms/float.py

```diff
@@ -13,24 +13,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-from typing import Union
+import typing
 
+from neptune.new.attributes.atoms.copiable_atom import CopiableAtom
+from neptune.new.internal.container_type import ContainerType
 from neptune.new.internal.operation import AssignFloat
 from neptune.new.types.atoms.float import Float as FloatVal
-from neptune.new.attributes.atoms.atom import Atom
 
+if typing.TYPE_CHECKING:
+    from neptune.new.internal.backends.neptune_backend import NeptuneBackend
 
-class Float(Atom):
-    def assign(self, value: Union[FloatVal, float, int], wait: bool = False):
+
+class Float(CopiableAtom):
+    @staticmethod
+    def create_assignment_operation(path, value: float):
+        return AssignFloat(path, value)
+
+    @staticmethod
+    def getter(
+        backend: "NeptuneBackend", container_id: str, container_type: ContainerType, path: typing.List[str]
+    ) -> float:
+        val = backend.get_float_attribute(container_id, container_type, path)
+        return val.value
+
+    def assign(self, value: typing.Union[FloatVal, float, int], wait: bool = False):
```

Comment:
From what I see we could produce generic `CopiableAtom.assign` which will first invoke overriden `verifyValue` method and then enqueue assigment operation since `create_assignment_operation` is overriden.

Reply:
That's true, but not sure if it's worth it tbh. 
With `verify_value` a separate def (with one use case) would still have to be implemented for each subclass, and we'd be saving two lines of code per subclass (`with lock / self._enqueue...`).

I've separated out `create_assignment_operation` mainly because it's used both in `assign` and in `CopyAttribute.resolve`.